### ISSUE TITLE
想问一下WN18 和 WN18RR是什么关系

### ISSUE THREAD

**Issue by xuefengDevelop** [thunlp/OpenKE#76]
我是KG的小白，我想问一下WN18 和WN18RR 是什么关系？为什么WN18RR的mean rank， hit@10 relatively low。这个是来做sparse graph的测试吗？

提前感谢～

best

**Comment by ShulinCao** [comment]
WN18 and FB15k suffer from test leakage through inverse relations: a large number of test triples can be obtained simply by inverting triples in the training set. For example, the test set frequently contains triples such as (s, hyponym, o) while the training set contains its inverse (o, hypernym, s). To create a dataset without this property, FB15k-237 was introduced – a subset of FB15k where inverse relations are removed. And similarly, WN18 was corrected by WN18RR.

**Comment by xuefengDevelop** [comment]
Thank you for your reply, Shulin

It is very helpful!

Thanks again!

**Comment by jweihe** [comment]
thanks  a lot
